docs/upgrade/upgrade_to_v0.18.0.md

```diff
@@ -0,0 +1,97 @@
+# Upgrade to v0.18.0
+
+There are breaking changes between OpenPAI v0.14.0 and v0.18.0. Before v0.18.0, OpenPAI was based on Yarn and Kubernetes, and data was managed by HDFS. In v0.18.0, it has switched to a pure Kubernetes-based architecture. Many new features, such as `AAD authorization`, `Hivedscheduler`, `Kube Runtime`, `Marketplace`, etc., are also included in this release.
```

Comment:
Before v0.18.0 or before v0.14.0?

Reply:
One more comment to this, is this doc also work for <0.14 --> 0.18 upgrade && >0.14 to 0.18 upgrade?

Reply:
> One more comment to this, is this doc also work for <0.14 --> 0.18 upgrade && >0.14 to 0.18 upgrade?

Only for < 0.14 to 0.18. Since the intermediate releases are internal release, I think there is no need to include them in this doc.

Reply:
> Before v0.18.0 or before v0.14.0?

Before v0.18.0 is OK, I think. Or it could be "v0.14.0 and before v0.14.0".